clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleNetworkLoadBalancerCachingAgent.groovy

```diff
@@ -45,7 +45,7 @@ import static com.netflix.spinnaker.clouddriver.google.cache.Keys.Namespace.INST
 import static com.netflix.spinnaker.clouddriver.google.cache.Keys.Namespace.LOAD_BALANCERS
 
 @Slf4j
-class GoogleLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implements OnDemandAgent {
+class GoogleNetworkLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implements OnDemandAgent {
```

Comment:
I think it would make this pr easier to review if you pull out the renaming of `GoogleLoadBalancerCachingAgent` to `GoogleNetworkLoadBalancerCachingAgent` and submit those changes first.